code/game/objects/items/weapons/autolathe_disk/blackshield_disks.dm

```diff
@@ -54,6 +54,14 @@
 		/datum/design/autolathe/ammo/kurtz_lethal,
 		/datum/design/autolathe/ammo/kurtz_rubber,
 		/datum/design/autolathe/ammo/kurtz_practice = 0,
+		/datum/design/autolathe/ammo/sbaw_empty = 0,
+		/datum/design/autolathe/ammo/sbaw_beanbag,
+		/datum/design/autolathe/ammo/sbaw_pellet,
+		/datum/design/autolathe/ammo/sbaw_slug,
+		/datum/design/autolathe/ammo/m12empty = 1,
```

Comment:
why is this 1 but the BETTER sbaw is 0?

Reply:
The SBAW isn't better, this is for the Saiga, which takes drums, the drums have double its capacity, this was what Moon told me to do >:|

Reply:
Something along the lines of "Drums are straight up upgrades" (so they shouldn't be free)

Also people already print practice mags for 0 license cost, empty them, and refill them with normal ammo so not to spend points, so instead of that I'd rather give the option of "free" empty mags (they still cost metal and such)

Reply:
Im thinking old stats for it were it was 10 in sbaw and 8 in the m12